MAINTAINERS.md

```diff
@@ -2,14 +2,18 @@
 
 Firecracker is maintained by a dedicated team within Amazon:
 
-- Adrian Catangiu <acatan@amazon.com>
-- Alexandru Cihodaru <cihodar@amazon.com>
-- Alin Dima <alindima@amazon.com>
-- Andrei Sandu <sandreim@amazon.com>
+- Alexandra Iordache <aghecen@amazon.com>
+- Alexandru-Cezar Sardan <alsardan@amazon.com>
+- Andrei-Cristian Traistaru <atc@amazon.com>
+- Anton Fabian Patras <patrsap@amazon.com>
+- Babis Chalios <bchalios@amazon.com>
 - Diana Popa <dpopa@amazon.com>
-- Gabriel Ionescu <gbi@amazon.com>
-- George Pisaltu <gpl@amazon.com>
-- Iulian Barbu <iul@amazon.com>
-- Liviu Berciu <lberciu@amazon.com>
+- Jonathan Woollett-Light <jcawl@amazon.com>
 - Luminita Voicu <lumivo@amazon.com>
-- Serban Iorga <seriorga@amazon.com>
+- Marco Cali <xmarcalx@amazon.com>
+- Matthew Schlebush <schlebus@amazon.com>
+- Nikita Kalyazin <kalyazin@amazon.com>
```

Comment:
at least my email is not right. Should be amazon.co.uk.

Reply:
In the old maintainers file, all the email were with `.com` instead of appropriate local prefixes, which is why I stuck to that. Afaik we can receive e-mail to `<username>@amazon.com` even if we are not US-based. 

I don't know how much we want to disclose about the geographical location of our team members, but if that's not a concern I am happy to change this!

Reply:
You are right, just tested that I can receive on amazon.com.